highlight.io/pages/docs/[[...doc]].tsx

```diff
@@ -117,8 +117,9 @@ const useIntersectionObserver = (setActiveId: (s: string) => void) => {
 		}
 
 		const observer = new IntersectionObserver(callback, {
-			rootMargin: ' -5% 0px -90% 0px',
-			threshold: 0.9,
+			root: document.body,
```

Comment:
adding `document.body` as the root may break the sidebar on desktop. It can be completely omitted from the options instead

Does the right sidebar still work on desktop?